### PR TITLE
Fix doctest failure and expand doctest coverage

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Needed for a clean build in https://github.com/dwavesystems/dwave-ocean-sdk/pull/69
```
Doctest summary
===============
   15 tests
    0 failures in tests
    0 failures in setup code
    0 failures in cleanup code
```

### CODE REVIEW

File: docs/intro.rst [hunk 1]
```diff
@@ -102,13 +103,14 @@ it sets a weak ``chain_strength`` to show broken chains.
...
 
     # Sample with low chain strength
-    sampleset = sampler.sample(bqm, num_reads=1000, chain_strength=0.1)   # doctest: +SKIP
+    sampleset = sampler.sample(bqm, num_reads=1000, chain_strength=0.1)
+
+Inspect the problem::
 
-    # Inspect
-    dwave.inspector.show(sampleset)    # doctest: +SKIP
+    dwave.inspector.show(sampleset)
```
[the remark this answers is not in the document]
Comment (JoelPasvolsky) [reply docs/intro.rst:113]: Yes, the # doctest: +SKIP does not work in testcode and I wanted to test at least what could be tested

**Comment by JoelPasvolsky** [comment]
The inspector is not tested. My understanding is that the returned `sampleset` also contains stuff that inspector looks for in its `.show()` but because now it's getting a `sampleset` from MockDWaveSampler those things won't be there, so it fails.

**Comment by randomir** [comment]
Ok, thanks. I'll look into that as a separate issue.

**Comment by JoelPasvolsky** [comment]
Here's an error message example for you:
```
Exception raised:
    Traceback (most recent call last):
      File "c:\Python36\Lib\doctest.py", line 1330, in __run
        compileflags, 1), test.globs)
      File "<doctest default[0]>", line 15, in <module>
        dwave.inspector.show(sampleset)    # doctest: +SKIP
      File "C:\Users\jpasvolsky\!git_ADTT\dwave-inspector\dwave-inspector\dwave\inspector\__init__.py", line 180, in show
        data = from_objects(*args, **kwargs)
      File "C:\Users\jpasvolsky\!git_ADTT\dwave-inspector\dwave-inspector\dwave\inspector\adapters.py", line 795, in from_objects
        "invalid combination of arguments provided: if data capture not "
    ValueError: invalid combination of arguments provided: if data capture not enabled, problem/response/solver/etc have to be explicitly specified
```

**Pull request merged**